scripts/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+$(npm bin)/babel \
```

Comment:
I use webpack for server-side code mainly for two reasons: better module isolation & CSS Modules. In this case babel is pretty straight, and I don't see profits of adding webpack layer. Is there any?